source/tests/utils_test.h

```diff
@@ -79,7 +80,7 @@ make_delay_message(const std::string& vj_uri,
     // start_date is used to disambiguate trips that are very late, cf:
     // https://github.com/CanalTP/chaos-proto/blob/master/gtfs-realtime.proto#L459
     trip->set_start_date(start_date);
-    trip->set_schedule_relationship(effect);
+    trip->set_schedule_relationship(transit_realtime::TripDescriptor_ScheduleRelationship_SCHEDULED);
```

Comment:
I think we should remove it if possible (or compute it from effect but...)

Reply:
Yes you've right, I can remove it now